XCUITests/FxScreenGraph.swift

```diff
@@ -483,6 +486,15 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.press(app.tables["History List"].cells.element(boundBy: 2), to: HistoryPanelContextMenu)
         screenState.tap(app.cells["HistoryPanel.recentlyClosedCell"], to: HistoryRecentlyClosed)
         screenState.tap(app.buttons["Done"], to: HomePanelsScreen)
+        screenState.gesture(forAction: Action.ClearLastHourHistory) { userState in
+            app.tables["History List"].staticTexts["Clear Recent History…"].tap()
```

Comment:
I've seen there is an id for this option, `identifier: 'HistoryPanel.clearHistory'` Please use it instead of the label.
In FxScreenGraph lets try to use ids whenever it is possible